src/App/store/AppState.ts

```diff
@@ -23,7 +23,7 @@ const dict = <T = any>(list: T[], classType?: { new (item: any): T }, key = 'nam
   }, {});
 };
 
-const isVideo = _.memoize((ext: string): boolean => ['mpeg4', 'mp4', 'webm'].includes(ext.toLowerCase()));
+const isVideo = _.memoize((ext: string): boolean => ['mp4', 'webm', 'mov', 'avi'].includes(ext.toLowerCase()));
```

Comment:
This includes `avi` but the allowed extensions only includes the first 3. I'm guessing that won't cause an issue but seems inconsistent.  However, is there a reason to need to include `avi`?

Reply:
avi renders because of this line, but it doesn't show up in the preview. So I thought I would disable it from being selected but leave it in here so that it could still be rendered. If that reasoning makes sense.

Reply:
Yep, happy with that.